src/browser/navigation-panel.js

```diff
@@ -13,18 +13,24 @@ const {Element} = require('./element');
 const {navigateTo, showTabStrip, focus} = require('./actions');
 const {KeyBindings} = require('./keyboard');
 const url = require('./util/url');
+const {readTheme} = require('./theme');
 
-const WindowControls = Component(options =>
-  DOM.div({className: 'windowctrls'}, [
+const WindowControls = Component(({webViewer}) => {
+  const theme = readTheme(webViewer);
+
+  return DOM.div({className: 'windowctrls'}, [
     DOM.div({className: 'windowctrl win-close-button',
+             style: {backgroundColor: theme.close},
```

Comment:
I was mainly interested in `readTheme` function so I would not have to say `style:{backgroundColor: theme.close}` but instead say a `style: theme.closeButton`, so that if in a future if we decide to style more than just `backgroundColor` we would have to only update `readTheme` function.